libavcodec/vvc/vvcdsp.h

```diff
@@ -172,16 +172,22 @@ typedef struct VVCDSPContext {
     VVCALFDSPContext alf;
 } VVCDSPContext;
 
-void ff_vvc_dsp_init(VVCDSPContext *hpc, int bit_depth);
+void ff_vvc_dsp_init(VVCDSPContext *hpc, int bit_depth,
```

Comment:
IMO it would be more elegant to pass `VVCFrameParamSets`, `VVCFrameContext` or something else more general here (and the call site for `ff_vvc_dsp_init` supports it), however including vvc_ps.h in vvcdsp.h introduced a lot of compilation errors.